examples/sfc/erc20bridge/java/src/main/java/net/consensys/gpact/examples/sfc/erc20bridge/ERC20TokenBridgeExample.java

```diff
@@ -64,34 +64,16 @@ public static void main(String[] args, boolean blockchainBIsMassConservation) th
     Credentials erc20OwnerCreds = CredentialsCreator.createCredentials();
     MassConservationERC20Bridge chainA =
         new MassConservationERC20Bridge(
-            "ChainA",
-            BigInteger.valueOf(CHAIN_A_TOKEN_SUPPLY),
-            erc20OwnerCreds,
-            root.bcId,
-            root.blockchainNodeRpcUri,
-            root.gasPriceStrategy,
-            root.period);
+            "ChainA", BigInteger.valueOf(CHAIN_A_TOKEN_SUPPLY), erc20OwnerCreds, root);
     AbstractERC20Bridge chainB;
     if (blockchainBIsMassConservation) {
       chainB =
           new MassConservationERC20Bridge(
-              "ChainB",
-              BigInteger.valueOf(CHAIN_B_TOKEN_SUPPLY),
-              erc20OwnerCreds,
-              bc2.bcId,
-              bc2.blockchainNodeRpcUri,
-              bc2.gasPriceStrategy,
-              bc2.period);
+              "ChainB", BigInteger.valueOf(CHAIN_B_TOKEN_SUPPLY), erc20OwnerCreds, bc2);
     } else {
       chainB =
           new MinterBurnerERC20Bridge(
-              "ChainB",
-              BigInteger.valueOf(CHAIN_B_TOKEN_SUPPLY),
-              erc20OwnerCreds,
-              bc2.bcId,
-              bc2.blockchainNodeRpcUri,
-              bc2.gasPriceStrategy,
-              bc2.period);
+              "ChainB", BigInteger.valueOf(CHAIN_B_TOKEN_SUPPLY), erc20OwnerCreds, bc2);
```

Comment:
is the `tokenSupply` required (or meaningful) for `MinterBurnerERC20Bridge` or only for `MassConservationERC20Bridge`? If not, should we remove this field from the abstract class and instead have it in the relevant subclass?

Reply:
This field has been removed from the MinterBurner